### ISSUE TITLE
react/no-unknown-property: allow viewbox on marker

### ISSUE THREAD

**Issue by dylang** [jsx-eslint/eslint-plugin-react#3416]
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox

As of `7.31.7`:

```jsx
error: Invalid property 'viewBox' found on tag 'marker', but it is only allowed on: svg (react/no-unknown-property):
  18 |     <Styled.BubbleMeter height={Math.round(radius * 2)} width={Math.round(radius * 2)} role="presentation">
  19 |       <defs>
> 20 |         <marker id={markerId} viewBox="0 0 2 2" refX="1" refY="1" markerWidth="1" markerHeight="1" orient="auto">
     |                               ^
  21 |           <Styled.Cap cx="1" cy="1" r="1" palette={capPalette} strokeWidth="0" />
  22 |         </marker>
  23 |       </defs>
```
